core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -82,6 +82,16 @@ public ColumnVisibility convert(String value) {
     }
   }
 
+  /**
+   * A catch all for older legacy options that have been dropped.  Most of them were replaced with
+   * accumulo-client.properties in 2.0.  Others have been dropped completely.
+   */
+  @Parameter(names = {"-p", "-tc", "--tokenClass", "-i", "--instance", "--debug", "-fake",
+          "--mock", "--site-file", "--keytab", "--ssl",  "--sasl"},
+          description = "Legacy options that have been dropped. Use accumulo-client.properties for" +
+                  "any connection or token options. See '-c, --config-file' option.")
+  private String legacyOpts = null;
```

Comment:
Should things fail if this is not null?  Would that be hard to do (every command has to check it)?

Reply:
I "think" we can get most of the commands with this class.  I was thinking throw and error with some useful breadcrumbs if the user tried to use a legacy option.

Reply:
It looks like this will only be useful for client CLI options.  Any other server side CLI stuff should be OK (Shell, Admin utils, etc).  The biggest change was with the client properties file which is where all these properties now reside.